server/routes/api.js

```diff
@@ -24,13 +24,37 @@ router.get('/add/:id', (req, res, next) => {
     });
 });
 
+router.get('/prefill/:climberID/:boulderID', (req, res, next) => {
+  const { climberID, boulderID } = req.params;
+  Climbed_boulder.findOne({
+    climber_id: climberID,
+    boulder_id: boulderID,
+  })
+    .then(data => {
+      data === null
+        ? res.send({
+            _id: '',
+            climber_id: '',
+            boulder_id: '',
+            date: '',
+            projected: false,
+            attempts: '',
+            result: '',
+            liked: false,
+            level_feedback: 0,
+          })
+        : res.send(data);
```

Comment:
I think it is a good practice to send a server status here as well.

Reply:
I have added it there and all the other places where it was missing.